### PR TITLE
add pwtools

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pwtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - matplotlib
```
Comment (xhochy) [inline recipes/pwtools/meta.yaml:25]: Can you  `matplotlib-base` here to avoid the Qt dependency?
```suggestion
    - matplotlib-base
```

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/pwtools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [not linux or py2k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+    - numpy
```
Comment (beckermr) [inline recipes/pwtools/meta.yaml:25]: Why is numpy in the build section?

File: recipes/pwtools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
+  run:
+    - python
+    - matplotlib-base
+    - nose
+    - numpy
```
Comment (beckermr) [inline recipes/pwtools/meta.yaml:38]: If you are lining against numpy you need to use a pin compatible statement here per the docs.

File: recipes/pwtools/meta.yaml (hunk 2 above)
Comment (jan-janssen) [reply recipes/pwtools/meta.yaml]: I removed it.

Comment (jan-janssen) [reply recipes/pwtools/meta.yaml]: I removed it.

File: recipes/pwtools/meta.yaml (hunk 3 above)
Comment (jan-janssen) [reply recipes/pwtools/meta.yaml:37]: No we use f2py to compile Fortran code, but there is no direct link to numpy, instead blas and lapack are used directly.

**Pull request merged**